### DISCUSSION TITLE
No choice of currency in Expense Accounts

### DISCUSSION THREAD

**Discussion by rrikesh** [firefly-iii/firefly-iii#4688] [Firefly III]
I am running Firefly III version 5.5.8

I've noticed that there's no choice of currency when creating an Expense account:
![Screenshot 2021-04-19 at 21 51 40](https://user-images.githubusercontent.com/3250906/115280847-6eeb2e80-a159-11eb-9104-6a96b346729c.png)

I would expect it to have a dropdown for currency like for an asset account:
![Screenshot 2021-04-19 at 21 52 31](https://user-images.githubusercontent.com/3250906/115280963-8cb89380-a159-11eb-8cd3-1e638f770cd8.png)

Debug information generated at 2021-04-19 21:53:01 Indian/Mauritius for Firefly III version **5.5.8**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.5.8 |
| Firefly III API | 1.5.2 |
| PHP | 8.0.3 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 32788f8d-c37a-4adb-a670-8ce0b4d9046b |
| Using docker? | true |
| Telemetry | false |
| Layout | v1 |
| System TZ | Indian/Mauritius |
| Browser TZ | Indian/Mauritius |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | mysql |
| DB version | 16 (exp. 16) |
| Login provider | eloquent  |
| Trusted proxies (.env) |  |

| User info | Value |
| --- | --- |
| Session start | 2021-04-01 00:00:00 |
| Session end | 2021-04-30 23:59:59 |
| Session first | 2021-01-01 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:87.0) Gecko/20100101 Firefox/87.0

**Comment by JC5** [comment]
That's right, there isn't.

**Reply by JC5** [reply]
(converted to discussion)

**Reply by rrikesh** [reply]
Is it something intentional?

**Reply by JC5** [reply]
Technically, you don't need a currency preference for any account. It's all stored in the transactions, not the accounts.

It does make it easier to create transactions and it helps maintain the internal consistency of the DB. But on the whole the system could do without.

For expense and revenue accounts, their balance is not relevant and their currency even less so. Any transaction in any currency can be sent to such an account.

**Reply by rrikesh** [reply]
Thanks for the update. I have now disabled Euro so that my accounts get my default currency automatically. I'll see how it works in the future.

![Screenshot 2021-04-20 at 09 43 45](https://user-images.githubusercontent.com/3250906/115343404-e9996580-a1bc-11eb-8015-fdbb14fbfb98.png)
